packages/navigation-menu/src/lib/navigation-menu/components/menu/menu.component.scss

```diff
@@ -0,0 +1,86 @@
+@import "../../assets/styles.scss";
```

Comment:
there's a mix of tabs / spaces here which messes up indenting. Seen in at some other places to. Maybe use or install linting to fix this permanently